src/test/java/edu/kis/powp/visitor/VisitorTest.java

```diff
@@ -0,0 +1,48 @@
+package edu.kis.powp.visitor;
+
+import edu.kis.powp.jobs2d.command.OperateToCommand;
+import edu.kis.powp.jobs2d.command.SetPositionCommand;
+import edu.kis.powp.jobs2d.command.visitor.CommandVisitor;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class VisitorTest {
```

Comment:
Please, add test to application, which counts "current command" (see DriverCommandManager)

Reply:
I actually meant extending the application with another test taken from the menu.